test/models/bounced_mail_address_test.rb

```diff
@@ -0,0 +1,104 @@
+require 'test_helper'
+
+class BouncedMailAddressTest < ActiveSupport::TestCase
+  include ActionMailer::TestHelper
+
+  def setup
+    @bounced_mail = BouncedMailAddress.new
+    @bounced_mail.email = 'recipient@registry.test'
+    @bounced_mail.message_id = '010f0174a0c7d348-ea6e2fc1-0854-4073-b71f-5cecf9b0d0b2-000000'
+    @bounced_mail.bounce_type = 'Permanent'
+    @bounced_mail.bounce_subtype = 'General'
+    @bounced_mail.action = 'failed'
+    @bounced_mail.status = '5.1.1'
+    @bounced_mail.diagnostic =  'smtp; 550 5.1.1 user unknown'
+  end
+
+  def test_email_is_required
```

Comment:
There is no need to test if field is required - we are set it presence to be validated and validation is already tested in rails.